docs/tutorials/up12_installation.rst

```diff
@@ -0,0 +1,45 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _up12_installation_tutorial:
+
+.. |propeller_name| replace:: UP12
+.. |motor_name| replace:: Vertiq 2306
+
+*********************************************
+Installing |propeller_name| on a |motor_name|
+*********************************************
+.. figure:: ../_static/tutorial_images/up12_installation/annotated_propeller.png
+    :align: center
+    :width: 750
+    :alt: Annotated |propeller_name|
+
+    Annotated |propeller_name|
+
+To install the |propeller_name| on a |motor_name| module you must first remove the main shoulder bolt from the adapter. Be sure not to lose the two washers on either side of the propeller hub. 
+
+.. figure:: ../_static/tutorial_images/up12_installation/hub_removed.jpg
+    :align: center
+    :width: 500
+    :alt: Hub removed from adapter
+
+    Adapter and Hub Separated
+
+Use the M2x6 Screws to mount the adapter to the |motor_name|.
```

Comment:
Did we supply the M2s?  If we did, this should be "Use the supplied M2x6 screws..."

If we didn't, it should be "Use 4 (it looks like it's 4 anyway, if it's not replace it with the right number) M2x6 Screws ..."